doc/develop/beyond-GSG.rst

```diff
@@ -401,6 +401,58 @@ Depending on the topology of build infrastructure, it may be preferable to simpl
 the ``/opt/zephyr/sdk/$VERSION`` directory to neighboring build machines. Alternatively, use
 ``/opt/zephyr/sdk/$VERSION`` to create packages using the package manager of choice.
 
+.. _gs_package_managers:
+
+Guidelines for Package Managers
```

Comment:
This sounds more like it should be part of the SDK documentation -- at minimum, it should not be part of the "Getting Started Guide."